### ISSUE TITLE
Black screen after wake

### ISSUE THREAD

**Issue by Edoardo001** [Edoardo001/Matebook-13-Hackintosh#16]
Since updated catalina 10.15.4 after i put to sleep, when i reopen lit or click to wake the laptop, screen remain black(backlight turn on and keys backlit too), but i can ear audio so laptop is working. Trying with external monitor, external monitor work after sleep, laptop remain black with backlight as the problem.
Injected same ig platform that was worked on mojave and tryied some other, edid injected, backlight working. Don't know what to try.

I didn't test sleep con Catalina 10.15.3 because of no time, so i don't remember if was working. Someone can't test if it works on 10.15.3 or 10.15.4? this drive me crazy, hope to find a fix in next days

**Comment by Saigoxx** [comment]
I solved this problem, replace the boot file with this one. Everything is normal with him, and I don’t see such a problem anymore.  EFI/BOOT/
https://github.com/Saigoxx/OLEG/blob/master/BOOTX64.efi

**Comment by Edoardo001** [comment]
Ok fixed, it was a bug of old version of clover.

**Comment by Pedroooow** [comment]
Hi, 

I have a similar issue after having switched to Opencore. 
Do you experience the same ? 

Symptoms are : when I let the laptop open go to sleep on its own (like when I leave my desk for a few hours), when I come back the laptops screen stays black, keys are backlit, and fans are spining a lot, laptop is very hot. 

the only way to avoid it is to manually put it to sleep before leaving.

**Comment by Edoardo001** [comment]
@Pedroooow Are you using latest version?

**Comment by Pedroooow** [comment]
Was on 1.4, swapped for 1.5, no improvement still having this issue

**Comment by Edoardo001** [comment]
Have you tryied command to fix in my guide(under improvements)? In dortania guide you will find a way to check what wake up your laptop

**Comment by Pedroooow** [comment]
yes I used your commands in the guide,  however I don't see any improvements. 
I checked with dortania's guide, here's what I've got 
![Capture d’écran 2020-07-02 à 19 11 12](https://user-images.githubusercontent.com/31337572/86390228-fd102a00-bc97-11ea-96fc-b02dc3c1a2ff.png)

**Comment by Edoardo001** [comment]
To fix blackscreen after sleep, try add this boot arg: igfxonln=1
if persist please tell me

**Comment by Pedroooow** [comment]
Ok seems to have solved it, I'll test it for the next few days and let you know ! 

Thanks :)
